app/views/unattended/provisioning_templates/bootdisk/generic_efi_host.erb

```diff
@@ -0,0 +1,76 @@
+<%#
+name: Boot disk Grub2 EFI - generic host
+snippet: false
+model: ProvisioningTemplate
+kind: Bootdisk
+description: |
+  Boot disk Grub2 EFI - generic host
+-%>
```

Comment:
Don't we need to add requirements for bootdisk plugin in order to be able actually use these templates? Something like https://github.com/theforeman/foreman/blob/develop/app/views/unattended/report_templates/host_-_applicable_errata.erb#L20

Reply:
You are right, thank you!